### ISSUE TITLE
New test failure in ament_package

### ISSUE THREAD

**Issue by clalancette** [ament/ament_package#55]
The nightly builds had a new test failure in ament_package, i.e. http://ci.ros2.org/job/nightly_win_rel/471/ .  It's a little hard to see the output there, so I ran the tests on ament_package locally with `src/ament/ament_tools/scripts/ament.py test --build-tests --symlink-install --isolated --only ament_package`, and saw:

```
checking ./test/test_pep257.py
1 errors
F
======================================================================
FAIL: test.test_pep257.test_pep257
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/ubuntu/ros2_ws-again/src/ament/ament_package/test/test_pep257.py", line 21, in test_pep257
    assert rc == 0, 'Found docblock style errors'
AssertionError: Found docblock style errors
```

**Comment by clalancette** [comment]
Oh, I didn't see the actual error above:

```
./ament_package/package.py:86 in public method `get_build_type`: D205: 1 blank line required between summary line and description (found 0)
```

I'll open a PR to fix this.

**Comment by mikaelarguedas** [comment]
fixed in #54
